base/sysinfo.jl

```diff
@@ -200,7 +212,19 @@ function _cpu_summary(io::IO, cpu::AbstractVector{CPUinfo}, i, j)
     end
     println(io)
 end
+"""
+    Sys.cpu_summary()
+
+Prints a summary of CPU information, organizing and displaying aggregated data for CPUs with the same model.
 
+# Arguments
```

Comment:
The style guide says to provide a separate `# Arguments` section only for complex functions with many arguments, which isn't the case here.

Instead, document the two arguments in prose, usually in the first sentence.